### ISSUE TITLE
Question: Uploads of audio + podcasts

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#3321]
Audio is obviously a major content type on almost every platform. This has been discussed at length in various areas, but given how long development has been ongoing I don't know what the answer to this is just yet:

1. Is uploading an audio file (such as an .mp3) currently possible on Joystream?
2. Does Atlas support this?

This feature has been requested by a new user here: https://github.com/Joystream/atlas/issues/2191 and it would be good to have an authorative answer (I searched but couldn't find one)

**Comment by bedeho** [comment]
The challenge here is not infrastructure: chain & storage & content delivery subsystems don't care.

The challenge is what to do in Atlas. Presumably what is being proposed here is genuine audio files, not video with an audio track, in this case, that is a very different product experience - something much more like Spotify/Soundcloud, that would take time to build. That is not within scope for us before launch, and frankly its something the DAO should consider funding itself I think, rather than Jsgenesis team, as it's totally orthogonal to everything else, so someone should be able to do it well without any important accomodation from the rest of the system.

**Comment by kdembler** [comment]
Agreed that the chain doesn't care, although we probably should do #2739 so that we can have some kind of generic content record on-chain instead of calling everything a video. This should also help with identifying type of content
